lib/deps/clone.js

```diff
@@ -54,13 +39,19 @@ module.exports = function clone(object) {
     return cloneArrayBuffer(object);
   }
 
-  if (!isPlainObject(object)) {
-    return object;
+  if (typeof Blob !== 'undefined' && object instanceof Blob) {
+    return object.slice(0, object.size, object.type);
+  }
+
+  if (typeof Buffer !== 'undefined' && object instanceof Buffer) {
+    var copy = new Buffer(object.length);
+    object.copy(copy);
+    return copy;
```

Comment:
^ this will browserify all of `Buffer`. probably the best is to use a snippet of code with the `-browser` trick to exclude it. I'll submit a PR